minigalaxy/installer.py

```diff
@@ -167,6 +170,35 @@ def copy_thumbnail(game):
     return error_message
 
 
+def create_applications_file(game):
+    error_message = ""
+    preferences_switch = Config.get("create_applications_file")
+    if game.platform == "linux" and preferences_switch:
+        path_to_shortcut = os.path.join(APPLICATIONS_DIR, game.name+".desktop")
+        # Create desktop file definition
+        desktop_context = {
+            "game_bin_path": os.path.join(game.install_dir, 'start.sh'),
```

Comment:
We support way more methods to launch the game (for multiple platforms, etc.) please add at the top of the fail (import section)
"
from minigalaxy.laucher import get_execute_command
"

And change this line to:
"
"game_bin_path": get_execute_command(game),
"

Reply:
Done, now it uses the function instead of start.sh.